Orchestrator/docs/NLRModels.md

```diff
@@ -1,19 +1,69 @@
+It is a 12-layer pretrained pretrained [Transformer][7] model optimized for conversation.
+Its architecture is pretrained for example-based use ([KNN][3]), thus it can be used out of box.
+
+### pretrained.20201210.microsoft.dte.00.12.unicoder_multilingual.onnx
+This is a high quality multilingual base model for intent detection.
+It is a 12-layer pretrained pretrained [Transformer][7] model optimized for conversation.
+Its architecture is pretrained for example-based use ([KNN][3]), thus it can be used out of box.
+
+### pretrained.20210105.microsoft.dte.00.12.bert_example_ner_multilingual.onnx (preview)
+This is a high quality multilingual base model for entity extraction.
+It is a 12-layer pretrained pretrained [Transformer][7] model optimized for conversation.
+Its architecture is pretrained for example-based use ([KNN][3]), thus it can be used out of box.
+
+### pretrained.20210105.microsoft.dte.00.12.tulr_example_ner_multilingual.onnx (preview)
+This is a high quality multilingual base model for entity extraction.
+It is a 12-layer pretrained pretrained [Transformer][7] model optimized for conversation.
+Its architecture is pretrained for example-based use ([KNN][3]), thus it can be used out of box.
+
+### pretrained.20210205.microsoft.dte.00.06.bert_example_ner.en.onnx (preview)
+This is a high quality EN-only base model for entity extraction. It's smaller and faster than its 12-layer alternative.
+It is a 6-layer pretrained pretrained [Transformer][7] model optimized for conversation.
+Its architecture is pretrained for example-based use ([KNN][3]), thus it can be used out of box.
+
+### pretrained.20201210.microsoft.dte.00.06.unicoder_multilingual.onnx
+This is a high quality multilingual base model for intent detection. It's smaller and faster than its 12-layer alternative.
+It is a 6-layer pretrained pretrained [Transformer][7] model optimized for conversation.
+Its architecture is pretrained for example-based use ([KNN][3]), thus it can be used out of box.
+
+### pretrained.20210205.microsoft.dte.00.06.bert_example_ner_multilingual.onnx (preview)
+This is a high quality multilingual base model for entity extraction. It's smaller and faster than its 12-layer alternative.
+It is a 6-layer pretrained pretrained [Transformer][7] model optimized for conversation.
+Its architecture is pretrained for example-based use ([KNN][3]), thus it can be used out of box.
+
+### pretrained.20210205.microsoft.dte.00.06.tulr_example_ner_multilingual.onnx (preview)
+This is a high quality multilingual base model for entity extraction. It's smaller and faster than its 12-layer alternative.
+It is a 6-layer pretrained pretrained [Transformer][7] model optimized for conversation.
+Its architecture is pretrained for example-based use ([KNN][3]), thus it can be used out of box.
 
 ## Models Evaluation
 For a more quantitative comparison analysis of the different models see the following performance characteristics.
```

Comment:
Need to add the new models to the valuation table

Reply:
Will let Ning create another PR on valuation.